resources/public/api/conf/3.0/schemas/retrieve_self_employment.json

```diff
@@ -419,9 +419,9 @@
               "example": "1000.45"
             },
             "capitalAllowanceSingleAssetPool": {
-              "description": "Capital Allowances for single asset pool costs. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+              "description": "Claim to capital allowances in respect of all single asset pools. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
               "type": "number",
-              "example": "1000.45"
+              "example": "500.25"
```

Comment:
Theres one more `capitalAllowanceSingleAssetPool` that needs changing around line 966